src/it/resources/org/checkstyle/suppressionxpathfilter/importorder/SuppressionXpathRegressionImportOrderOne.java

```diff
@@ -0,0 +1,8 @@
+package org.checkstyle.suppressionxpathfilter.importorder;
+
+import static java.lang.Math.PI;
+import java.util.Set; //warn
```

Comment:
add a whitespace between `//` and `warn`, same for all other cases

Reply:
done